jep/14/README.adoc

```diff
@@ -0,0 +1,289 @@
+  In such cases these entries should be discussed in the entity channels and signed off by entity leaders.
+  _Roadmap maintainers_ still review the formal side of the request.
+* Maintainers of other plugins and components are eligible to submit their roadmap proposals.
+  Such pull requests will be reviewed by Roadmap Maintainers, see the _Eligibility_ section below.
+* Any other Jenkins community member can submit a proposal.
+  If there is no SIG/sub-project behind the proposal, it will likely require a discussion in Jenkins channels to verify eligibility and improve visibility of the proposal.
+
+==== Maintaining roadmap entries
+
+We expect the initiative submitters to track progress of the roadmap initiatives and to update the initiative statuses accordingly.
+The “Roadmap maintainers” team will be doing periodic scrubs of the roadmap to discover and modify outdated entries,
+but there is no guarantee of full consistency for the roadmap (“eventual consistency”).
+Whomever discovers an issue, pull requests are welcome.
+
+==== Roadmap review meeting
+
+Once per quarter _Roadmap maintainers_ will organize a public roadmap review meeting.
+It may be held as a part of the https://jenkins.io/project/governance-meeting/[Jenkins Governance Meeting] or as a separate meeting.
+This meeting will be used to discuss the completed initiatives, verify the state of the roadmap and to identify the missing initiatives.
+
+==== Preventing the roadmap from bloating
+
+_Roadmap maintainers_ are responsible to keep the number of items in the roadmap within a comprehensible range.
+Recommendation is to have a limited number of projects per category:
+
+* “Current” - up to 3
+* “Near Term” - up to 3
+* “Future” - up to 5
+
+== Rollout plan
+
+Once the proposal is confirmed in principle, the following process will be applied:
+
+* The proposal is implemented and submitted as a pull request
+* A developer mailing list email is sent to facilitate contributions to the draft (initial data and initiatives)
+* `Mar 25, 2020` - There is voting at the governance meeting which signs-off publishing of the roadmap draft
+* The roadmap draft is published as a draft with explicit “Work in Progress” disclaimers in the Web UI
+* All entities in Jenkins (SIGs, sub-projects, etc.) are contacted via mailing lists and invited to contribute to the roadmap draft
+* `2 weeks later` - There is a blogpost with a roadmap draft announcement and invitation to contribute in public
+* Roadmap draft is presented at the Jenkins online meetup for developers
+* `2+ weeks later` - Governance meeting with voting for publishing the roadmap as an active version
+* Work-in-progress disclaimers are removed, the roadmap is considered as official
+* Announcements in social media and the Jenkins blog
+
+After the rollout, the roadmap will be maintained by the _Roadmap maintainers_ team as documented above.
+
+== Motivation
```

Comment:
Almost 3 years later, I still do not understand what is the value of keeping _Motivation_ here instead of putting it right after Abstract

Reply:
The problem is for external readers who would benefit from motivation explaining the JEP's purpose.

Reply:
Like I said:

> You want to propose changing it? Go for it. 

I'm `-0` on the change.  If others agree with you, I'm fine with it.

Reply:
I think this JEP would benefit by having the motivation earlier in the document.  In all the discussions of roadmap, I had not understood the benefit of a published, easily navigated roadmap for new conitrbutors.  They are shown plans which link to more details about those plans and can more readily decide which of those plans are most interesting to them.

+1 for placing the motivation immediately after the abstract.

Reply:
Let's decouple it to a separate change anyway, more feedback is needed